torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -75,6 +75,14 @@ class TORCH_API TensorExprKernel {
     return graph_;
   }
 
+  std::vector<ConstantDescr> getConstantDescriptors() const {
+    return constants_;
+  }
+
+  std::vector<CodeGen::BufferArg> getBufferArgs() const {
+    return bufferArgs_;
```

Comment:
Should these return `const std::vector<>&`?

Reply:
probably yes :) good catch!